SConstruct

```diff
@@ -3,10 +3,12 @@ import distutils
 import os
 import subprocess
 import sys
+from sysconfig import get_paths
 
 zmq = 'zmq'
 arch = subprocess.check_output(["uname", "-m"], encoding='utf8').rstrip()
 
+python_path = get_paths()['include']
 # Rebuild cython extensions if python, distutils, or cython change
 cython_dependencies = [Value(v) for v in (sys.version, distutils.__version__, Cython.__version__)]
```

Comment:
this can be deleted now

Reply:
I still need to link with python.h to run .cpp files generated by cythonize

Reply:
I meant line 13

Reply:
Oh this is just for custom builders? I guess I can also remove it in openpilot then